operator-pipeline-images/operatorcert/schemas/ci-schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "title": "ci.yaml validation schema",
+    "description": "json schema validating the ci.yaml",
+    "type": "object",
+    "additionaProperties": true,
+    "properties": {
+        "fbc": {
+            "description": "Config set for FBC-enabled operator bundles",
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "description": "Whether the operator already have FBC support or not",
+                    "type": "boolean"
+                },
+                "catalog_mapping": {
+                    "description": "Catalog Mapping accoering to provided templates types",
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "template_name": {
+                                "description": "Name of the catalog template file (required)",
```

Comment:
What's the pattern for adding `(required)` to the description? It is a bit inconsistent. Ether removed it from the description since it is already marked as required in the schema or add it to all descriptions for all required fields.